dash/_callback.py

```diff
@@ -27,9 +29,9 @@ class NoUpdate:
     pass
 
 
-GLOBAL_CALLBACK_LIST = []
-GLOBAL_CALLBACK_MAP = {}
-GLOBAL_INLINE_SCRIPTS = []
+GLOBAL_CALLBACK_LIST: List[dict] = []
+GLOBAL_CALLBACK_MAP: Dict[str, dict] = {}
```

Comment:
If taken further, I'll try replace `dict` with `Dict` with more information about the underlying structure.